class-tgm-plugin-activation.php

```diff
@@ -926,7 +924,7 @@ public function dismiss() {
 		 * @param array $plugin Array of plugin arguments.
 		 */
 		public function register( $plugin ) {
-			if ( ! isset( $plugin['slug'] ) || ! isset( $plugin['name'] ) ) {
+			if ( empty( $plugin['slug'] ) || empty( $plugin['name'] ) ) {
```

Comment:
Excellent - I've actually taken notice that `empty()` does the exists check or not.